src/v/cloud_storage/tests/tx_range_manifest_test.cc

```diff
@@ -41,7 +41,17 @@ static remote_manifest_path
 
 using tx_range_t = model::tx_range;
 
-static std::vector<tx_range_t> ranges = {
+template<typename T = int>
+static fragmented_vector<T>
+make_fragmented_vector(std::initializer_list<T> in) {
+    fragmented_vector<T> ret;
+    for (auto& e : in) {
+        ret.push_back(e);
```

Comment:
Can one move out of a initializer_list?

Reply:
my hunch would be yes. but being a test helper, im not sure how much it matters?

Reply:
See #13854